packages/Uploader/src/components/File/File.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CheckIcon from "@paprika/icon/lib/Check";
+import TimesIcon from "@paprika/icon/lib/Times";
+import RetryIcon from "@paprika/icon/lib/Refresh";
+import Button from "@paprika/button";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import tokens from "@paprika/tokens";
+import { UploaderContext } from "../../Uploader";
+import { getNumberWithUnits } from "../../helpers";
+import statuses from "../../statuses";
+import * as div from "./File.styles";
+
+const propTypes = {
+  error: PropTypes.string,
+  fileKey: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  progress: PropTypes.number,
+  size: PropTypes.number.isRequired,
+  status: PropTypes.oneOf(Object.keys(statuses).map(key => statuses[key])).isRequired,
+};
+
+const defaultProps = {
+  error: "",
+  progress: 0,
+};
+
+function File({ error, fileKey, name, progress, size, status }) {
+  const uc = React.useContext(UploaderContext);
+  const { cancelFile, restartFileUpload } = uc;
+  const I18n = useI18n();
+
+  function getUploadedAmount() {
```

Comment:
return on one line with arrow function?

Reply:
...or just declare as a `const`?

Reply:
Yeah I dont know why I did that, lol. Fixing.